src/appleseed.python/studio/helpers.py

```diff
@@ -28,7 +28,8 @@ def convert_all_textures_to_tx(maketx_path):
         texture_parameters = texture.get_parameters()
         texture_path = texture_parameters['filename']
 
-        if texture_path.endswith('.tx'):
+        if texture.get_model() != 'disk_texture_2d' or texture_path.endswith('.tx'):
```

Comment:
we can split the two conditions in two tests. It's natural that we ignore non image textures, we should not log anything if get_model() is not a disk_texture_2d.